docs/release.md

```diff
@@ -0,0 +1,81 @@
+Release process
+===============
+
+The goal of this document is to define the release process of KubeVirt, but it
+is not intended to define any release criteria.
+
+
+Overview
+--------
+- KubeVirt uses [semantic versioning](http://semver.org)
+- Primary artefact is the source tree in form of a signed tap using
+  [git evtag](https://github.com/cgwalters/git-evtag)
+- Binary artifacts are built using automation
+- The releases appear on a time based release schedule
+
+
+Content
+-------
+The primary artifact of a release is the source tree itself. The trust on the
+tree is established by using _git-evtag_ which can be used to sign the tree
+recursively, including blobs and submodules.
+
+For convenience a number of binary artifacts can be provided alongside a
+release, those are:
+
+- Container images (currently docker images)
+- Client sided binaries (i.e. _virtctl_)
+
+
+Cadence
+-------
+A release is taking place on every first Monday of a month.
+The release owner is in charge of delaying a release if it is _really_
+necessary.
+If a release has to be delayed by more than a week, then the release must be
```

Comment:
Would be "for more" better?

Reply:
Just a matter of taste according to a native speaker :)